docs/dg/dev/backend-development/cronjobs/multi-process-run-console.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Multi process run console
+description: Describes the MultiProcessRunConsole command for long-running worker processes or repeated console command execution in a controlled and restartable way.
+keywords: MultiProcessRunConsole, Jenkins, Command, Console, Worker.
+last_updated: Nov 10, 2025
+template: howto-guide-template
+related:
+  - title: Cronjobs
+    link: docs/dg/dev/backend-development/cronjobs/cronjobs.html
+  - title: Jenkins stability best practices
+    link: docs/ca/dev/best-practices/best-practises-jenkins-stability.html
+  - title: Jenkins operational best practices
+    link: docs/ca/dev/best-practices/jenkins-operational-best-practices.html
+---
+
+## Overview
+
+The `MultiProcessRunConsole` command is designed for long-running worker processes or repeated console command execution in a controlled and restartable way.
+
+It allows you to:
+
+- Automatically restart console commands in the Jenkins job.
+
+- Run commands in a continuous loop with a defined total execution time limit.
+
+- Execute commands in separate processes to prevent shared memory or cache collisions.
+
+- Enforce child process execution timeouts.
+
+- Gracefully stop via signals (SIGTERM, SIGQUIT).
+
+## Installing the `MultiProcessRunConsole`
+
+1. Require the package:
+
+```bash
+composer require spryker/console:"^4.16.0"
+```
+
+2. Add a console command to `src/Pyz/Zed/Console/ConsoleDependencyProvider.php`:
+
+```php
+<?php
+
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Console\Communication\Plugin\Console\MultiProcessRunConsole;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    protected function getConsoleCommands(Container $container): array
+    {
+        $commands = [
+            // ...
+            new MultiProcessRunConsole(),
+        ];
+    }
+}
+```
+
+3. Configure in `config/Zed/cronjobs/jenkins.php`:
+
+```php
+/* Example of usage multi-process:run in jenkins config */
+$jobs[] = [
+    'name' => 'multi-process-update-product-label-relations',
+    'command' => '$PHP_BIN vendor/bin/console multi-process:run \"product-label:relations:update -vvv --no-touch\" 20 --separate_thread',
```

Comment:
I still don't get, why " should be escaped?
According to PHP notation it's useless.

```suggestion
    'command' => '$PHP_BIN vendor/bin/console multi-process:run "product-label:relations:update -vvv --no-touch" 20 --separate_thread',
```